### PR TITLE
[WIP] Handlers now receive an additional parameter 'data'

### PR DESCRIPTION
**Pull request by itsdeka**
Handlers now receive an additional parameter 'data' containing all raw ws data. 
It can be used to retrieve extra flags if enabled.

Please see https://github.com/bitfinexcom/bitfinex-api-py/issues/140

Additionals:
-) Fixed errors in README.md
-) Adjusted all WS examples
-) Updated version and changelog

### PR status:
- [X] Version bumped
- [X] Change-log updated

### CODE REVIEW

File: bfxapi/examples/ws/multiple_instances.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ async def do(self):
                 create_instances(instances_to_create)
                 break
 
-        to_unsub = [f"t{ticker}" for ticker in subbed_tickers if ticker in get_random_list_of_tickers()]
+        to_unsub = [f"t{ticker}" for ticker in subbed_tickers if f"t{ticker}" in get_random_list_of_tickers()]
```
Comment (robertkowalski) [inline bfxapi/examples/ws/multiple_instances.py:103]: unrelated fix?

Comment (itsdeka) [reply bfxapi/examples/ws/multiple_instances.py]: ye removed
